guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>Thanks John for authoring this wonderful guide, it has been nicely structured and comprehensive, in presenting a universally workable approach for cloning items from source to target portals. Very neat, thanks!! </p><p>I have raised some questions in the previous comments, please check!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>